.travis/prerelease_lobby_install

```diff
@@ -1,39 +0,0 @@
-#!/bin/bash
```

Comment:
The idea of spinning up a lobby server and doing some testing I think is a good one and in some ways a better idea than what these scripts were trying to do. The intent of the install lobby and update maps scripts were to automatically deploy maps and lobby components. The maps were to go to the bot servers and stay in sync automatically. The lobby would be deployed to a 'prerelease' lobby server that could be used for manual testing (so spinning up a self contained lobby for automated testing has strengths). I was never able to get the SSH tokens to work out quite well, so the scripts did not work to push artifacts to our server.